2023-12-17T01:33:26.998Z: Error: connect ETIMEDOUT 2400:8901::f03c:93ff:fe32:9cc3:443
    at AxiosError.from (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:517:28)
    at eventHandlers.<computed> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:517:28)
    at TLSSocket.socketErrorListener (node:_http_client:501:9)
    at TLSSocket.emit (node:events:517:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-17T01:57:36.246Z: Error: connect ETIMEDOUT 2400:8901::f03c:93ff:fe32:9cc3:443
    at AxiosError.from (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:517:28)
    at eventHandlers.<computed> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:517:28)
    at TLSSocket.socketErrorListener (node:_http_client:501:9)
    at TLSSocket.emit (node:events:517:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-17T02:00:01.230Z: Error: connect ETIMEDOUT 2400:8901::f03c:93ff:fe32:9cc3:443
    at AxiosError.from (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:517:28)
    at eventHandlers.<computed> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:517:28)
    at TLSSocket.socketErrorListener (node:_http_client:501:9)
    at TLSSocket.emit (node:events:517:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-17T02:00:33.592Z: Error: connect ETIMEDOUT 2400:8901::f03c:93ff:fe32:9cc3:443
    at AxiosError.from (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:517:28)
    at eventHandlers.<computed> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:517:28)
    at TLSSocket.socketErrorListener (node:_http_client:501:9)
    at TLSSocket.emit (node:events:517:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-17T02:04:47.827Z: Error: connect ETIMEDOUT 2400:8901::f03c:93ff:fe32:9cc3:443
    at AxiosError.from (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:517:28)
    at eventHandlers.<computed> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:517:28)
    at TLSSocket.socketErrorListener (node:_http_client:501:9)
    at TLSSocket.emit (node:events:517:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-17T02:11:03.162Z: Error: connect ETIMEDOUT 2400:8901::f03c:93ff:fe32:9cc3:443
    at AxiosError.from (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:517:28)
    at eventHandlers.<computed> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:517:28)
    at TLSSocket.socketErrorListener (node:_http_client:501:9)
    at TLSSocket.emit (node:events:517:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-17T02:11:15.799Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: folder is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:76:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-17T02:13:08.550Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: folder is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:76:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-19T04:57:00.091Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-12-19T05:00:00.280Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at runNextTicks (node:internal/process/task_queues:64:3)
    at listOnTimeout (node:internal/timers:538:9)
    at process.processTimers (node:internal/timers:512:7)
2023-12-19T05:00:30.831Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-12-19T06:57:00.065Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-12-19T07:00:00.309Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-12-19T07:00:30.139Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-12-20T06:57:01.018Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-12-20T07:00:25.507Z: AxiosError: Request failed with status code 508
    at settle (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:1967:12)
    at IncomingMessage.handleStreamEnd (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3062:11)
    at IncomingMessage.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-20T09:00:48.662Z: Error: read ECONNRESET
    at AxiosError.from (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:514:28)
    at eventHandlers.<computed> (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:514:28)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-24T03:06:17.752Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: pm2 is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:145:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-24T03:06:22.466Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: pm2 is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:145:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-24T03:06:27.488Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: pm2 is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:145:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-24T03:06:32.501Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: pm2 is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:145:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-24T03:06:37.475Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: pm2 is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:145:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-24T03:06:42.495Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: pm2 is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:145:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-24T03:06:47.527Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: pm2 is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:145:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-24T03:09:50.757Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: pm2 is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:145:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-24T03:09:55.618Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: pm2 is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:145:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-24T03:10:00.626Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: pm2 is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:145:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-24T03:31:01.252Z: Error: Unhandled Rejection at: [object Promise]. Reason: ReferenceError: number is not defined
    at process.<anonymous> (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:529:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-25T11:08:40.208Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-12-25T11:08:40.229Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-12-25T11:08:40.419Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at runNextTicks (node:internal/process/task_queues:64:3)
    at process.processTimers (node:internal/timers:509:9)
2023-12-25T11:08:40.520Z: Error: Protocol error (Runtime.callFunctionOn): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:218:35)
    at ExecutionContext._evaluateInternal (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:204:50)
    at ExecutionContext.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:27)
    at DOMWorld.evaluate (C:\Users\valen\OneDrive\Desktop\WaFresh\finalbotwa\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:97:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-12-27T05:14:40.099Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:17:49.365Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:17:51.367Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:17:53.397Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:17:55.444Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:17:57.443Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:17:59.471Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:18:01.452Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:18:03.469Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:18:05.493Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:18:07.507Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:21:04.333Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:21:06.319Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:21:08.400Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:21:10.386Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:21:12.419Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:21:14.447Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:21:16.460Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:21:18.475Z: Error: Unhandled Rejection at: [object Promise]. Reason: Error: net::ERR_INTERNET_DISCONNECTED at https://web.whatsapp.com/
    at process.<anonymous> (C:\Users\jaja.valentino\Desktop\finalbotwa\production-wa.js:77:14)
    at process.emit (node:events:526:35)
    at emit (node:internal/process/promises:150:20)
    at processPromiseRejections (node:internal/process/promises:284:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
2023-12-27T05:57:37.403Z: AxiosError: Request failed with status code 508
    at settle (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:1967:12)
    at IncomingMessage.handleStreamEnd (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3062:11)
    at IncomingMessage.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-27T07:21:58.926Z: Error: getaddrinfo ENOTFOUND srs-ssms.com
    at AxiosError.from (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:514:28)
    at eventHandlers.<computed> (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:514:28)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-29T05:00:11.196Z: AxiosError: Request failed with status code 508
    at settle (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:1967:12)
    at IncomingMessage.handleStreamEnd (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3062:11)
    at IncomingMessage.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-12-29T07:03:29.075Z: AxiosError: Request failed with status code 508
    at settle (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:1967:12)
    at IncomingMessage.handleStreamEnd (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3062:11)
    at IncomingMessage.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2024-01-15T07:57:52.972Z: Error: connect ETIMEDOUT 192.53.117.183:443
    at AxiosError.from (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:514:28)
    at eventHandlers.<computed> (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:514:28)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2024-01-15T08:00:44.940Z: Error: connect ETIMEDOUT 192.53.117.183:443
    at AxiosError.from (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:514:28)
    at eventHandlers.<computed> (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:514:28)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2024-01-15T08:03:43.158Z: Error: connect ETIMEDOUT 192.53.117.183:443
    at AxiosError.from (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:514:28)
    at eventHandlers.<computed> (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:514:28)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2024-01-15T08:05:43.045Z: Error: connect ETIMEDOUT 192.53.117.183:443
    at AxiosError.from (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3083:25)
    at RedirectableRequest.emit (node:events:514:28)
    at eventHandlers.<computed> (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:514:28)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2024-01-17T07:04:22.505Z: AxiosError: Request failed with status code 508
    at settle (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:1967:12)
    at IncomingMessage.handleStreamEnd (C:\Users\jaja.valentino\Desktop\finalbotwa\node_modules\axios\dist\node\axios.cjs:3062:11)
    at IncomingMessage.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
